nbs/07_nlp.external.transformers-glue.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Can you explain why you have a <code>SortedDL</code>  here? (I know why, but many do not ;) )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/walkwithfastai/walkwithfastai.github.io/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>